draft-ietf-core-oscore-groupcomm.md

```diff
@@ -842,11 +855,15 @@ The value of the 'kid' parameter in the 'unprotected' field of response messages
 
 The value of the 'kid context' parameter in the 'unprotected' field of request messages MUST be set to the ID Context, i.e., the Group Identifier value (Gid) of the group. That is, unlike in {{RFC8613}}, the 'kid context' parameter is always present in requests.
 
-## AEAD Nonce # {#sec-cose-object-aead-nonce}
+## Nonce Computation # {#sec-cose-object-aead-nonce}
+
+The nonce is constructed like in OSCORE, with the difference that step 4 in {{Section 5.2 of RFC8613}} is replaced with:
 
-The AEAD nonce is constructed like in OSCORE, with the difference that step 4 in {{Section 5.2 of RFC8613}} is replaced with:
+A. and then XOR with the X bytes from the Common IV's start, where X is the length in bytes of the nonce.
 
-A. and then XOR with the X least significant bits of the Common IV, where X is the length in bits of the AEAD nonce.
+For example, if X = 7 and the Common IV is 0x00112233445566778899aabbcc (13 bytes), then the bytes to XOR are 0x00112233445566 (7 bytes).
+
+The constructed nonce is in fact used as AEAD nonce by the AEAD Algorithm (see {{ssec-common-context-aead-alg}}) and by the Group Encryption Algorithm when this is an AEAD algorithm (see {{ssec-common-context-cs-enc-alg}}).
```

Comment:
I would remove "in fact" and just put this as:
_The constructed nonce is used as AEAD nonce by the AEAD Algorithm (see {{ssec-common-context-aead-alg}}) and by the Group Encryption Algorithm when this is an AEAD algorithm (see {{ssec-common-context-cs-enc-alg}})._

Reply:
Fixed in https://github.com/core-wg/oscore-groupcomm/commit/486ca64666db862fd1e9bc0dc367818a6d8bcde7